Add support for "Omnipay: Square"

dfear opened the pull request:
<!--
Please check the following steps to submit your pull request. If you have any questions please read the contribution guide available at https://go.invoiceplane.com/contribguide or join the community forums or the Slack channel.
You can check items by changing `[ ]` to `[x]`.
If you can't check all checklist items please add `[WIP]` in front of your title.
Remove this first paragraph but please keep the following checklist even if it's incomplete. Pull requests without the checklist will be rejected.
-->

Pull Request Checklist

  * [x] My code follows the code formatting guidelines.
  * [WIP] I have an issue ID for this pull request. 
  * [x] I selected the corresponding branch.
  * [x] I have rebased my changes on top of the corresponding branch.
  
Issue Type (Please check one or more)

  * [ ] Bugfix
  * [ ] Improvement of an existing Feature 
  * [x] New Feature

nielsdrost7 commented:
The development issue ID is https://development.invoiceplane.com/browse/IP-764 feel free to log into development.invoiceplane.com and add more information to this feature.

What else did you do with this Omnipay feature? Besides adding it to codeigniter composer.json? Did you make extra functionality?

dfear commented:
> The development issue ID is https://development.invoiceplane.com/browse/IP-764 feel free to log into development.invoiceplane.com and add more information to this feature.
> 
> What else did you do with this Omnipay feature? Besides adding it to codeigniter composer.json? Did you make extra functionality?

1. added the required info to:
application/config/payment_gateways.php

2. added the necessary language strings to:
application/language/english/gateway_lang.php

P.S.
What else is required to add an Omnipay payment gateway?

naui95 commented:
Omnipay is being removed from InvoicePlane. We are starting to interact directly with the gateways, which is also possible for square (https://developer.squareup.com/us/en). I'm closing this PR, since it cannot be implemented anymore.
